Sources/Common/Service/CIOApiEndpoint.swift

```diff
@@ -8,6 +8,7 @@ public enum CIOApiEndpoint {
     case trackCustomerEvent(identifier: String)
     case pushMetrics
     case trackDeliveryMetrics
+    case trackNSEPushDeliveryMetrics
```

Comment:
Do we need to introduce a new case here? I see `trackDeliveryMetrics` in the line above that I was expecting we would use instead.

Reply:
Wanted to keep the current implementation for CIO tracking APIs separated from CIO data pipeline APIs. And all other cases would get removed because we don't use track API endpoints anymore.

Reply:
Ok, I understand why adding another case makes sense. 

```suggestion
    case trackPushMetricsCdp
```

I think it would be good to use a name that describes the endpoint (track push metrics), instead of a name that describes the caller (NSE).